HISTORY.md

```diff
@@ -1,5 +1,80 @@
 # Version history
 
+## 1.3.0-rc.1 /2020-12-01
```

Comment:
I suggest to add `v1.3.0 [final]` release notes. :)

Reply:
I prefer to do first a release candidate. If everything is going well, then we will release the final version.